rpc/glfs-operations.c

```diff
@@ -119,6 +119,71 @@ glusterBlockCheckAvailableSpace(struct glfs *glfs,
 }
 
 
+int
+glusterBlockZeroFill(struct glfs_fd *tgfd, off_t offset, size_t size)
+{
+  struct iovec iov[4];
+  char *zerodata = NULL;
+  size_t chunksize = 4194304;  //4MiB
```

Comment:
I see above not changing anywhere, should it be #define instead? Anyways, compiler takes care of optimizing, would be good to comment on why it is a 'variable' (if there is a reason).

Reply:
@amarts make-sense to me, I wrote the code which chunks this initial assignment by 4 buffers, but later I ruled-out that code, and forget to change this.

Will update this in the next spin. Thanks!